### DISCUSSION TITLE
Footnote numbers too hard to see on jump

### DISCUSSION THREAD

**Discussion by asmeurer** [pradyunsg/furo#581] [Ideas]
When you click on the number of a footnote to jump back to its location in the original text, it can be hard to see where exactly the footnote in question is. One issue is that it is at the top of the page, but also the footnote styling is a small font size. It also can fall behind the "back to top" thing (https://github.com/pradyunsg/furo/discussions/435) which seems to appear when clicking one of these links.

It would help to highlight the footnote somehow when the page anchor is `#id1`, `#id2`, etc. A suggestion would be to put a blue box around the footnote number, which I think I've seen done on other sites.

**Comment by asmeurer** [comment]
Just noticed that it does highlight the footnote number with a background color. But it is so small that it's impossible to see this unless you zoom in on the page.

**Comment by pradyunsg** [comment]
I've been playing around with this and... honestly, open to ideas / examples of how other websites handle this!

**Comment by asmeurer** [comment]
I actually found that 

```py
html_theme_options = {
    "color-highlight-on-target": "var(--color-highlighted-background)",
}
```

is not insignificant improvement to the *status quo* here. Namely, this makes the highlighting blue rather than yellow, which is almost impossible to see for such a small footnote number text (for my docs, the blue actually works better with my color theming than yellow in all instances; but you may want to just change this for the `footnote-reference` class).

Perhaps relatedly, I also found that

```css
/* Highlight footnotes when they are linked to */
.footnote:target {
    background-color: var(--color-highlight-on-target);
    color: var(--color-foreground-primary);
}
```

is really useful. This causes the footnote itself to be highlighted when it is linked to. Otherwise, you have to remember what number the footnote was. Since the footnotes are at the bottom of the page, you can't just judge which footnote is the current target by whatever is at the top of the viewport. Another option is to put a blue border around the footnote, which is what GitHub does.[^footnote] I'm not sure what CSS is used to do that. 

[^footnote]: Click this footnote to see what I mean.
